lib/faktory_worker/error_formatter.ex

```diff
@@ -0,0 +1,68 @@
+defmodule FaktoryWorker.ErrorFormatter do
```

Comment:
How did you determine this was the best logic to format the error, is this based on anything else or trial and error?

Reply:
Mostly based on trial and error with a range of different ways I could find to crash the job. This makes a best effort to mimic the way the other faktory clients send failures to faktory and then falls back to trying to provide as much info as possible.